TransactionMobile/TransactionMobile/Presenters/LoginPresenter.cs

```diff
@@ -139,7 +162,7 @@ private async Task GetConfiguration()
         {
             try
             {
-                //this.LoginViewModel.EmailAddress = "merchantuser@testmerchant3.co.uk";
+                //this.LoginViewModel.EmailAddress = "merchantuser@testmerchant1.co.uk";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/MobileApplication/pullRequest?prid=5700460)